typing/includeclass.ml

```diff
@@ -20,7 +20,14 @@ open Types
 let class_types env cty1 cty2 =
   Ctype.match_class_types env cty1 cty2
 
-let class_type_declarations env cty1 cty2 =
+let class_type_declarations ~loc env cty1 cty2 =
+  Builtin_attributes.check_deprecated_inclusion
+    ~def:cty1.clty_loc
+    ~use:cty2.clty_loc
+    loc
+    cty1.clty_attributes cty2.clty_attributes
+    (Path.last cty1.clty_path);
+
```

Comment:
Nitpick: I find this blank line makes it hard to see what is actually in `class_type_declarations`. I think leaving blank lines in the middle of definitions is against the unwritten style guide of the compiler.